### PR TITLE
[SPARK-15352][Doc] follow-up: add configuration docs for topology-aware block replication

### PR DESCRIPTION
**Pull request by lins05**
## What changes were proposed in this pull request?

Add configuration docs for topology-aware block replication

## How was this patch tested?

Generating the docs and preview it.

### CODE REVIEW

**Comment by lins05** [comment]
@shubhamchopra @cloud-fan PTAL

**Comment by lins05** [comment]
ping @shubhamchopra @cloud-fan

**Comment by shubhamchopra** [comment]
@lins05 Apologies for the delay in responding and thanks for adding the docs. LGTM.

**Comment by lins05** [comment]
Thanks @shubhamchopra. Could you please help review & merge this one @cloud-fan ?

**Comment by gatorsmile** [comment]
cc @cloud-fan

File: docs/configuration.md [hunk 1]
```diff
@@ -1004,14 +1004,48 @@ Apart from these, the following properties are also available, and may be useful
...
+  <td>
+    org.apache.spark.storage.<br />DefaultTopologyMapper
+  </td>
+  <td>
+    The topology information of a host is determined by a topology mapping service defined by the
+    abstract class <code>org.apache.spark.storage.TopologyMapper</code>, which can be configured by
+    this property. A default implementation that assumes all hosts are in the same rack is provided
+    by <code>org.apache.spark.storage.DefaultTopologyMapper</code>. A file-based implementation is
+    provided by <code>org.apache.spark.storage.FileBasedTopologyMapper</code>, which reads the
+    topology information from the file <code>org.apache.spark.storage.topologyFile</code>. Each line
```
Comment (cloud-fan) [inline docs/configuration.md]: shall we also add an entry for `org.apache.spark.storage.topologyFile`?

File: docs/configuration.md [hunk 2]
```diff
@@ -1004,14 +1004,48 @@ Apart from these, the following properties are also available, and may be useful
...
+    of this file is of the format of <code>host1 = /rack1</code> and provides a mapping from a host
+    name to its rack information. <em>Note:</em> This configuration only takes effect when
+    <code>spark.storage.replication.policy</code> is set to a a policy that takes the topology
```
Comment (cloud-fan) [inline docs/configuration.md]: nit: double `a`

**Comment by HyukjinKwon** [comment]
Hi @lins05, is this PR active?

**Pull request closed without merging**